### PR TITLE
Create legislators csv from votes data

### PR DESCRIPTION
**Pull request by schneidy**
Created a script to pull legislators' names from vote data and place it into a new CSV.

### CODE REVIEW

File: scripts/archive_leg_to_csv.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+    from opencivicdata.legislative.models import Bill, PersonVote
+    bills = Bill.objects.filter(
+        legislative_session__identifier=session,
+        legislative_session__jurisdiction_id=jurisdiction_id).values_list("id", flat=True)
+    for bill in bills:
+        voters = PersonVote.objects.filter(
+            vote_event_id__bill_id=bill
+        )
+        for voter in voters:
```
Comment (jamesturk) [inline scripts/archive_leg_to_csv.py]: check out collections.Counter for things like this

File: scripts/archive_leg_to_csv.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+            }
+        },
+        MIDDLEWARE_CLASSES=(),
+    )
+    django.setup()
+
+
+@click.command()
+@click.argument("state_abbr", nargs=-1)
+@click.argument("session", nargs=1)
```
Comment (jamesturk) [inline scripts/archive_leg_to_csv.py]: looking good, it'd be great to have the ability to run this for all sessions and spit out a file per session

File: scripts/archive_leg_to_csv.py [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    conf.settings.configure(
+        conf.global_settings,
+        SECRET_KEY="not-important",
+        DEBUG=False,
+        INSTALLED_APPS=(
+            "django.contrib.contenttypes",
+            "opencivicdata.core.apps.BaseConfig",
+            "opencivicdata.legislative.apps.BaseConfig",
+        ),
+        DATABASES={
```
Comment (jamesturk) [inline scripts/archive_leg_to_csv.py]: let's pull this from the environment like we do in to_database.py

Comment (jamesturk) [reply scripts/archive_leg_to_csv.py]: actually, let's refactor here and move init_django into utils maybe so we can just share it between the two

Comment (schneidy) [reply scripts/archive_leg_to_csv.py]: Moved into utils.

File: scripts/archive_leg_to_csv.py (hunk 2 above)
Comment (schneidy) [reply scripts/archive_leg_to_csv.py]: Script will now generate CSVs for all of a state's sessions if none provided.

File: scripts/archive_leg_to_csv.py (hunk 1 above)
Comment (schneidy) [reply scripts/archive_leg_to_csv.py]: Updated to use Counter

**Comment by jamesturk** [comment]
pushed a few commits to this branch to fix some issues

**Comment by jamesturk** [comment]
relates to openstates/openstates#3102

**Comment by jamesturk** [comment]
@schneidy I realized I forgot to mention another important piece of this. 

We also want to include names included in the BillSponsorship in the matching.  Can you pull those into the same sheet for the state?  We don't need to differentiate between occurrences as a voter and as a sponsor, but any BillSponsorships that don't have a matched person_id should be included as well. 

(BTW- I've pushed some changes to the branch, make sure you pull those first :))

File: scripts/archive_leg_to_csv.py [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    voter_dictionary = Counter()
+
+    bills = Bill.objects.filter(
+        legislative_session__identifier=session,
+        legislative_session__jurisdiction_id=jurisdiction_id,
+    ).values_list("id", flat=True)
+    for bill in bills:
+        voters = PersonVote.objects.filter(vote_event_id__bill_id=bill)
+
+        for voter in voters:
```
Comment (jamesturk) [inline scripts/archive_leg_to_csv.py]: I just noticed, this considers everyone unmatched, we only need to count someone in this sheet if they don't have a person_id assigned

Comment (schneidy) [reply scripts/archive_leg_to_csv.py]: Query now set to:

`voters = PersonVote.objects.filter(vote_event_id__bill_id=bill, voter_id=None)`

**Comment by schneidy** [comment]
> @schneidy I realized I forgot to mention another important piece of this.
> 
> We also want to include names included in the BillSponsorship in the matching. Can you pull those into the same sheet for the state? We don't need to differentiate between occurrences as a voter and as a sponsor, but any BillSponsorships that don't have a matched person_id should be included as well.
> 
> (BTW- I've pushed some changes to the branch, make sure you pull those first :))

BillSponsors are now included.

**Pull request closed without merging**